src/platform/intel/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 # SPDX-License-Identifier: BSD-3-Clause
-
+if(CONFIG_CAVS)
 add_subdirectory(cavs)
+endif()
```

Comment:
I think these CMakeLists.txt files are only needed for XTOS builds, so for Zephyr-only platforms you shouldn't care about them

Reply:
BTW the fastest way to debug build issues is to... break the build. Life is too short to read verbose CMake logs; lesson learned from reading too many of them.

For instance in this particular case try this:

```diff
  # SPDX-License-Identifier: BSD-3-Clause
+ BREAK THE BUILD NOW
  add_subdirectory(cavs)
```

If this compiles fine with Zephyr then you can be pretty confident this `if()` is not required.